src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -52,16 +51,17 @@ public static boolean isChimeric(GKInstance rle) throws Exception {
         return true;
     }
 	
+	// TODO: If this method is being heavily used by a QA check, consider using 'fetchInstance(48887L)`. It's much quicker than a global DB search.
```

Comment:
This is an important comment, and since it's at the method-level, could you please upgrade it to Javadoc? I think my "TODO" idea was about leaving a note to implement a cached instance so that there is only ever one database query.